Makefile

```diff
@@ -18,6 +21,11 @@ include Makefile.*.mk
 # More info on the awk command:
 # http://linuxcommand.org/lc3_adv_awk.php
 
+.PHONY: generate-docs
```

Comment:
@pipo02mix This whole Makefile will get overwritten by "Align files" PRs again (see top hint about the file being generated by devctl). We need to either [mark it with the right flavor](https://github.com/giantswarm/github/blob/main/repositories/team-phoenix.yaml) (as done for `cluster-aws`, for instance) or pray that we'll soon merge `cluster-aws`+`default-apps-aws` into one repo so that this doesn't get reverted by mistake.

Reply:
ok thx, then I will add it as a flavour. Indeed I need to make it part of a workflow to automate the run once a new release is done. Maybe it will fit better as GH action. WDYT?